docs-2.0/2.quick-start/4.nebula-graph-crud.md

```diff
@@ -236,12 +236,11 @@ You can use the `INSERT` statement to insert vertices or edges based on existing
 
 ## Read data
 
-You can use the statements [MATCH](../3ngql-guide/../3.ngql-guide/7.general-query-statements/2.match.md), GO (doc TODO), FETCH (doc TODO), and [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) to retrieve vertex and edge data.
-
-* `MATCH` is the most commonly used, powerful, and flexible statement for graph data querying. It relies on native indexes and patterns to match data stored in Nebula Graph.
 * The `GO` statement traverses the database based on specific conditions. A `GO` traversal starts from one or more vertices, along one or more edges, and return information in a form specified in the `YIELD` clause.
 * `FETCH` is used to get properties from vertices or edges.
-* The `LOOKUP` statement is based on [indexes](#about_indexes). It is used together with the `WHERE` clause to search for the data that meet the specific conditions.
+* The [LOOKUP](../3.ngql-guide/7.general-query-statements/5.lookup.md) statement is based on [indexes](#about_indexes). It is used together with the `WHERE` clause to search for the data that meet the specific conditions.
+* [MATCH](../3ngql-guide/../3.ngql-guide/7.general-query-statements/2.match.md) is the most commonly used statement for graph data querying. It relies on native indexes and patterns to match data stored in Nebula Graph.
+    > **CAUTION:** Indexes can **reduce** the writing performance by 90% or even more. **DO NOT** use indexes in production environments unless you are fully aware of their influences on your service.
```

Comment:
Improper choosing indexes can

Reply:
Correct use of indexes may also significantly reduce the performance, right?